include/capture_util.h

```diff
@@ -0,0 +1,18 @@
+#ifndef GUARD_CAPTURE_UTIL_H
+#define GUARD_CAPTURE_UTIL_H
+
+struct BallData
+{
+    u16     multiplier;
+    u16     divider;
+    bool8   guaranteedCapture;
+    u8      flatBonus;
+};
+
+u32 ComputeCaptureOdds();
```

Comment:
```suggestion
u32 ComputeCaptureOdds(void);
```
As-written, `ComputeCaptureOdds` takes any arguments (this behavior changes in C23, but we're `-std=gnu17`).